sdk/tests/features/feature_extractor.py

```diff
@@ -0,0 +1,52 @@
+import importlib
+import inspect
+import pkgutil
+
+from features.get_project_root import get_project_root
+
+
+def validate_feature_list(feature_list):
+    for feature in feature_list:
+        if feature == "":
```

Comment:
i think it makes more sense to put the validation in the decorator itself to which checks for empty strings